projects/azure/README.md

```diff
@@ -251,6 +251,7 @@ we will not need to have the rest of the secrets in place.
   --from-file=cluster/test-deploy/azure/ssh-privatekey \
   --from-file=cluster/test-deploy/azure/certs.yaml \
   --from-file=cluster/test-deploy/azure/.dockerconfigjson \
+  --from-file=cluster/test-deploy/azure/redhat-io-docker-config.json \
```

Comment:
I think you also need to update `cluster/test-deploy/azure/.gitignore` to ignore the new secret.

Reply:
Updated

Reply:
I also added some more missing secrets we recently added